gen-core/src/lib.rs

```diff
@@ -92,7 +92,7 @@ impl<'de> serde::Deserialize<'de> for HashId {
 impl fmt::Display for HashId {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         for byte in &self.0 {
-            write!(f, "{:02x}", byte)?;
+            write!(f, "{byte:02x}")?;
```

Comment:
Some clippy appeasement here

Reply:
yeah these are definitely better this way. Surprised it let me get by